packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -115,6 +115,17 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
             });
         }));
 
+        this.term.attachCustomKeyEventHandler(event => {
+            const ctrlCmd = event.ctrlKey || event.metaKey;
```

Comment:
You can improve this part. You need cover only Ctrl/Cmd + C and Ctrl/Cmd + V. But user could press few functional keys in the same time , for example: Ctrl + Alt + C. So, You should apply checks that Alt, Shift, Meta was not pressed. For example:
const ctrlCmd = (event.ctrlKey || event.metaKey) && !(event.shiftKey || event.metaKey || event.altKey)

Reply:
Good catch! Thanks